src/components/pages/FamilySettings/RenderFamilySettings.js

```diff
@@ -29,7 +29,7 @@ const FamilySettings = props => {
 
   return (
     <Layout className="parent-dashboard add-child">
-      <ParentNavSider selected="settings" />
+      <ParentNavTopBar selected="settings" />
```

Comment:
I notice 'selected' is passed as a prop with a different value every time you render the ParentNavTop but then I never see that value pulled through in the ParentNavTopBar component. What is this prop doing?

Reply:
I don't think that's a prop @Robert-Lark. I think that's setting up to show where a person is a page/component wise so that it can be displayed differently. However I could be wrong.

Reply:
Good eye! Thanks for catching that, guys. That is some relic code from the previous iteration of the navbar. I'll remove that and update the branch shortly.

Reply:
@brittanymclaughlin I think your right - my guess is that you can do something like adjust the color of the nav element corresponding to the current page you are on. This would allow the user to be able to see where in the navigation system they currently are. Not necessary here as it looks like the nav only has - log-out and change users but I think was should try to talk the stakeholders/UX into adding FAQ to the Nav in which case it would come in handy.